core/src/components/radio-button-group/radio-button-group.stories.js

```diff
@@ -84,17 +111,32 @@ export const Required = () => `
         <input type="radio" class="fudis-radio-button__input" name="fudis-fieldset-id" value="baz" />
         <div class="fudis-radio-button__content">
           <div class="fudis-radio-button__content-wrapper">
-            <div class="fudis-radio-button__content__control"></div>
+            <div class="fudis-radio-button__content__control">
+              <!-- When selected, render the indicator below -->
+              <!-- <span class="fudis-radio-button__content__control__indicator"></span> -->
+            </div>
           </div>
           <span class="fudis-radio-button__label">Option 3</span>
         </div>
       </label>
+      <!-- When invalid: add fudis-guidance__errors block or add a fudis-error-message block from below -->
```

Comment:
This example is missing `<div id="fudis-guidance-id" class="fudis-guidance">` block.